turborepo-tests/integration/tests/command-bin.t

```diff
@@ -0,0 +1,9 @@
+Setup
+  $ . ${TESTDIR}/../../helpers/setup.sh
+  $ . ${TESTDIR}/_helpers/setup_monorepo.sh $(pwd)
+
+  $ ${TURBO} bin -vvv > out.log
+  $ grep --quiet -E "Global turbo version: .*" out.log
+  $ grep --quiet -E "No local turbo binary found at" out.log
+  $ grep --quiet -E "Running command as global turbo" out.log
```

Comment:
Out of curiosity: is extended regex required for these patterns?

Reply:
no not in these cases, i've removed here. I think I was multi line editing these files and just used the same options